okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -124,7 +124,8 @@ private static RemotingOkHttpClient createInternal(
         TaggedMetricRegistry registry = DefaultTaggedMetricRegistry.getDefault();
 
         // Routing
-        UrlSelectorImpl urlSelector = UrlSelectorImpl.create(config.uris(), randomizeUrlOrder);
+        UrlSelectorImpl urlSelector = UrlSelectorImpl.createWithFailedUrlCooldown(config.uris(), randomizeUrlOrder,
+                config.failedUrlCooldown());
```

Comment:
I tried breaking this by replacing `config.failedUrlCooldown()` with `Duration.ZERO` and tests still seem to pass - can we try to add a test that would catch a bad refactor like this?

Reply:
`UrlSelectorImpl.create` calls `UrlSelectorImpl.createWithFailedUrlCooldown` with `Duration.ZERO`, which is equivalent to not caching.